pkg/drive/adaptor/storage/dummy.ts

```diff
@@ -7,6 +8,12 @@ export class LocalStorage implements Storage {
   constructor(private readonly basePath: string = './drive') {}
 
   async upload(name: string, file: Uint8Array) {
+    try {
+      await fs.statfs(this.basePath);
+    } catch (e) {
+      await fs.mkdir(this.basePath);
```

Comment:
```suggestion
      await fs.mkdir(this.basePath, { recursive: true });
```
こうしておかないと `basePath` が `hoge/foo` のようになっていた場合に再帰的に作成できないかと